client/packages/requisitions/src/RequestRequisition/DetailView/columns.ts

```diff
@@ -39,6 +39,24 @@ export const useRequestColumns = () => {
         getSortValue: rowData => rowData.item.name,
       },
     ],
+    {
+      key: 'unit',
+      label: 'label.unit',
+      align: ColumnAlign.Left,
+
+      // width: 100,
+      accessor: ({ rowData }) => rowData.item.unitName,
+      getSortValue: rowData => rowData.item.unitName ?? '',
+    },
+    {
+      key: 'defaultPackSize',
+      label: 'label.default-pack-size',
+      align: ColumnAlign.Left,
```

Comment:
numeric values should align right
```suggestion
      align: ColumnAlign.Right,
```